src/states/bootstrapping_peer.rs

```diff
@@ -279,61 +283,68 @@ impl Base for BootstrappingPeer {
         Transition::Stay
     }
 
-    fn handle_direct_message(
+    fn handle_message(
         &mut self,
-        msg: Variant,
-        p2p_node: P2pNode,
+        sender: Option<ConnectionInfo>,
+        msg: Message,
         _: &mut dyn EventBox,
-    ) -> Result<Transition, RoutingError> {
+    ) -> Result<Transition> {
+        let response = match msg.variant {
```

Comment:
Why the changes in this function? It seem we are still doing mostly the same thing?